integration-tests/gradle-plugin-test/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id("com.android.library") version "7.2.2" apply false
+    id("org.jetbrains.kotlin.android") version "1.7.10" apply false
```

Comment:
Should this version come from `buildSrc/Config.kt`? I think that is available here? 🤔

Reply:
I was intentionally trying to avoid the `buildSrc`-symlink hack, but since we also require linting-setup I just fell back on it to easily get access to all the versions and plugins 😞